### ISSUE TITLE
wireguard_autoupdate - syntax error on line 25 and line 36

### ISSUE THREAD

**Issue by chjohans** [vchrizz/ER-wizard-WireGuard#8]
Line 25 please change to:
 if [[ "$($wireguardwizard load | jq -r .data.pkgstatus)" =~ .*new\sversion\sfound.* ]]; then

Line 34 please change to:
if [[ "$($wireguardwizard load | jq -r .data.wizstatus)" =~ .*new\sversion\sfound.* ]]; then

Notice how the space in the string comparison is replaced with \s in the logical expression comparison part. As it was the script bugs out with a syntax error on my ERL with the latest firmware (2.0.9-hotfix2).

**Comment by vchrizz** [comment]
Thank you for your proposed change.
I'm not quite sure why you get a syntax error, could you please show the exact error you get or tell how to reproduce the issue?
For me the current code works and usually a literal space in a regular expression should work.

**Comment by chjohans** [comment]
I did, and now I no longer get any syntax errors. You're of course right that a literal spaced, like any liter al characters, is perfectly valid in a regular expression.

The \s is not the same though, as it will match any whitespace character. Not that that fact should matter here.
I did hours of debugging before I found that those two lines gave a syntax error on my system,m (ER Lite with 2.0.9-hotfix2). As strange as it might seem, after making those changes it worked just fine.

Now it's back working again, even without those changes, so I don't know what caused this and I see no point in digging deeper to find out either. The only thing I've changed on my side is that I've upgraded to 2.0.9-hotfix4.

Thanks for making this available!
